src/component/NodeEditor/NodeEditor.tsx

```diff
@@ -681,11 +1124,27 @@ export default class NodeEditor extends React.PureComponent<NodeEditorProps, Nod
     }
 
     private getSides(): Sides {
-        const action: AnyAction = getAction(this.props, this.state.config);
+        const action = getAction(this.props, this.state.config);
```

Comment:
i do see us refactoring this in the future to be more specific about which props should be passed to each form.